docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -0,0 +1,748 @@
+---
+title: Understanding role-based access control in Dagster Cloud | Dagster Docs
+---
+
+# Understanding role-based access control in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Role-based access control (RBAC) enables you to grant specific permissions to users in your organization, ensuring that the right people have appropriate access to what they require in Dagster Cloud, and no more.
+
+In this guide, we'll cover how RBAC works in Dagster Cloud, how to assign roles to users, and the granular permissions for each user role.
+
+---
+
+## Dagster Cloud user roles
+
+Dagster Cloud uses a hierarchical model for RBAC, meaning that the most permissive roles include permissions from the roles beneath them. The following user roles are currently supported, in order from the **most** permissive to the **least** permissive:
+
+- Organization Admin
+- Admin
+- Editor
+- Launcher (Enterprise plans only)
+- Viewer (Enterprise plans only)
+
+For example, the **Admin** user role includes permissions specific to this role and all permissions in the **Editor**, **Launcher**, and **Viewer** user roles. Refer to the [User permissions reference](#user-permissions-reference) for the full list of user permissions in Dagster Cloud.
+
+### User role enforcement
+
+All user roles are enforced both in Dagster Cloud and the [GraphQL API](/concepts/dagit/graphql).
+
+### Teams
+
+Dagster Cloud Enterprise users can create teams of users and assign default permission sets, either in Dagster Cloud or dynamically from an Identity Provider (IdP). Refer to the [Managing user teams in Dagster Cloud](/dagster-cloud/account/managing-users/managing-user-teams) guide for more info.
```

Comment:
To confirm: This IdP stuff - I haven't had a chance to check in on SCIM yet.

Reply:
we don't actually support setting the permissions in the iDP yet - you can map the people to teams there via SCIM, but which permissions a given team has is still solely controlled in the dagster cloud UI

Reply:
Thanks for clarifying! I'll update this